superdesk/core/resources/resource_rest_endpoints.py

```diff
@@ -594,47 +594,7 @@ def _populate_item_hateoas(
         }
 
         # extract related links from the `resource_config.data_class` relations (annotations)
-        related_links = self._get_related_links_from_annotations(item)
-        if related_links:
+        if related_links := self.resource_config.data_class.get_related_links(item):
```

Comment:
:+1:  Don't see `:=` too often

Reply:
I saw it once, long ago when it was proposed, then I forgot it was already there in python 3. I really like it, I think I will be using it more often 😃